cookbook/security/multiple_guard_authenticators.rst

```diff
@@ -0,0 +1,168 @@
+How to Use Multiple Guard Authenticators
+========================================
+
+Guard authentication component allows you to easily use many different authenticators at a time.
+
+An entry point is a service id (of one of your authenticators) whose start()
+method should be called when an anonymous user hits a page that requires authentication.
+
+Multiple authenticators with shared entry point
+-----------------------------------------------
+
+Let's have an example of two authenticators: one based on login form, another one on facebook login.
+Both authenticators entry points redirect user to the same login page.
+However, in your configuration you have to explicitly say which entry point you want to use.
+
+This is how your security configuration can look in action:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/security.yml
+        security:
+             # ...
+            firewalls:
+                default:
+                    anonymous: ~
+                    guard:
+                        authenticators:
+                            - app.form_login_authenticator
+                            - app.facebook_connect_authenticator
+                        entry_point: app.form_login_authenticator
+
+    .. code-block:: xml
+
+        <!-- app/config/security.xml -->
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <config>
+                <!-- ... -->
+                <firewall name="default">
+                    <anonymous />
+                    <guard entry_point="app.form_login_authenticator">
```

Comment:
should be `entry-point` (XML conventions are to not use underscores and the XmlFileLoader takes care of transforming all dashes into underscores)